spec/models/course_spec.rb

```diff
@@ -12,20 +14,36 @@
     it { is_expected.to validate_presence_of(:title) }
 
     context 'when course is created' do
-      subject { Course.new }
+      subject { Course.new(creator: owner, updater: owner) }
 
       it { is_expected.not_to be_published }
       it { is_expected.not_to be_opened }
+
+      it 'contains the provided user as the owner' do
+        expect(subject.course_users.map {|course_user| course_user.user}).to include(owner)
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Pass `&:user` as an argument to `map` instead of a block.